assets/css/style.css

```diff
@@ -6,13 +6,91 @@
     scroll-behavior: smooth;
     padding: 0;
     margin: 0;
-    font-family: Roboto;
+    font-family: 'Roboto', sans-serif;
+}
+
+a {
+    text-decoration: none;
 }
 
 .container {
-    padding: 0.01em 16px;
-    margin-top: 16px;
-    margin-bottom: 16px;
```

Comment:
Please add back the padding, margin-top, margin-bottom to avoid problems from other contributors which did not use GitHub.